### PR TITLE
docs(get-started) Simplify Get Started landing page

### PR DESCRIPTION
**Pull request by maggiehays**
The main function of this page is to be a landing spot after clicking "Get Started" on the docs home page:

CleanShot 2022-11-16 at 18 20 41

From there, we want end-users to be routed to appropriate resources based on their deployment method. 
CleanShot 2022-11-16 at 18 32 32

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Review by jjoyce0510 — APPROVED** [review]

File: docs/get-started-with-datahub.md [hunk 1]
```diff
@@ -1,63 +1,21 @@
...
-* [jq](https://stedolan.github.io/jq/download/)
-* [Docker Compose](https://github.com/docker/compose/blob/master/INSTALL.md) - if using Linux
-
-:::note
-
-Make sure to allocate enough hardware resources for Docker! 
-We recommend the following configuration: 2 CPUs, 8GB RAM, 2GB Swap area, and 10GB disk space.
-
-:::
+These guides are focused on helping you get up and running with DataHub as quickly as possible.
```
Comment (maggiehays) [inline docs/get-started-with-datahub.md:3]: What additional copy do we need here?

Comment (laulpogan) [reply docs/get-started-with-datahub.md:3]: Probably something detailing different entry points-- either pointing to ingestion, feature, or dev guides

File: docs/get-started-with-datahub.md [hunk 2]
```diff
@@ -1,63 +1,21 @@
 # Get Started With DataHub
 
-This section is focused on helping you get up and running with DataHub as quickly as possible.
-
-## Get Started with Self-Hosted DataHub
-
-This is a great place to start if you're new to DataHub and are interested in taking the open-source project for a spin.
```
Comment (laulpogan) [inline docs/get-started-with-datahub.md:7]: Per our convo: replace this with a brief "self-hosted datahub is the OSS implementation of our tool,etc.." just a small blurb about what you are clicking into, maybe a persona snippet like "if you're looking to build and deploy datahub yourself, start here" 

then BUTTON

File: docs/get-started-with-datahub.md [hunk 3]
```diff
@@ -1,63 +1,21 @@
...
     className='button button--primary button--lg'
+    style={{ margin: 12 }}
     href="quickstart">
     Get Started with Self-Hosted DataHub
 </a>
-</p>
-
-## Get Started with Managed DataHub
-
-[Acryl Data](https://www.acryldata.io/product) provides a fully managed, premium version of DataHub.
```
Comment (laulpogan) [inline docs/get-started-with-datahub.md:42]: Same thing here, you may not even need to change this line, just cut out the "you'll learn how too" and the "prereqs" and go straight to the button

**Pull request closed without merging**